src/cmd/team.go

```diff
@@ -53,9 +64,9 @@ var createMemberCmd = &cobra.Command{
 
 var contactType string
 var createContactCmd = &cobra.Command{
-	Use:        "contact TEAM_ID|TEAM_ALIAS ADDRESS DISPLAYNAME",
+	Use:        "contact {TEAM_ID|TEAM_ALIAS} ADDRESS DISPLAYNAME",
 	Short:      "Add a contact to a team",
-	Long:       `Add a contact to a team`,
+	Example:    `opslevel create contact --type=slack my-team #general General`,
```

Comment:
should we add more examples here?

Reply:
Maybe one more wouldn't hurt.  Or having all the types in the help command but that might be harder to do.

Reply:
Hmm ya i think i can do this